stac_fastapi/elasticsearch/setup.py

```diff
@@ -13,8 +13,8 @@
     "stac-fastapi.api==2.3.0",
     "stac-fastapi.extensions==2.3.0",
     "fastapi-utils",
-    "elasticsearch[async]",
-    "elasticsearch-dsl",
+    "elasticsearch[async]==7.17.1",
+    "elasticsearch-dsl==7.4.0",
```

Comment:
Doesn't this version pin run counter to the Elasticsearch v8 support mentioned in the PR title?

Reply:
Their recommended approach is to use the 7.x versions of these client libraries (elasticsearch-dsl isn't even out for 8) and pass the header `accept": "application/vnd.elasticsearch+json; compatible-with=7"`